.github/workflows/android.yml

```
@@ -89,7 +89,7 @@ jobs:
       - run: ls -lh dist/
       - name: start simulator
         run: ./ci/mac_start_emulator.sh
-      - run: bazel mobile-install --fat_apk_cpu=x86 --start_app //examples/kotlin/hello_world:hello_envoy_kt
+      - run: bazel mobile-install --host_force_python=PY2 --fat_apk_cpu=x86 //examples/kotlin/hello_world:hello_envoy_kt
```

```suggestion
      - run: bazel mobile-install --host_force_python=PY2 --fat_apk_cpu=x86 //examples/kotlin/hello_world:hello_envoy_kt
```
https://github.com/bazelbuild/bazel/issues/7899

---

No, that is just a red herring they already updated to pyhon 3 in https://github.com/bazelbuild/bazel/commit/d93a146463e20f4eedc5ae658b03ee0260750aa7#diff-947a45ac6fb178dbfb7cc530fddd6b9a